sr/robot/power_board.py

```diff
@@ -475,7 +476,7 @@ def __init__(self, serial: SerialWrapper):
         self._serial = serial
 
     @log_to_debug
-    def buzz(self, frequency: float, duration: float) -> None:
+    def buzz(self, frequency: float, duration: float, *, blocking: bool = False) -> None:
```

Comment:
TODO: Document this new argument

Reply:
bfbb2e36e3636eb9bf679f5664b74761d216df92